wordpress-mobile/WordPress-Android
# Issue/10644 material site creation

Fixes #10644

#11229 needs to be merged first.

This PR adds material theme and Dark Mode support to site creation flow.

The only change to logic is related to NEXT/SKIP buttons. Instead of styling a single button, we have two buttons and switch their visibility on demand.

It's kind of hard to take a screenshot of the shimmering skeleton layout, but I used the same colors as in the post list.

![comparison_site_creation](https://user-images.githubusercontent.com/728822/73820535-d90ef200-47a6-11ea-8c22-ff861ec20fea.png)

Landscape:
![comparison_cr_land](https://user-images.githubusercontent.com/728822/73820740-3c008900-47a7-11ea-8173-85e0ee424800.png)


To test:
- Go through site creation flow, and make sure that everything looks good in both light and dark themes.
Pay attention to Skip/Next buttons.

You can change the app theme from App Settings.

PR submission checklist:
